tests/accounts/test_integration.py

```diff
@@ -75,17 +75,15 @@ def test_creates_new_account_for_the_given_user(self):
 
 class TestCreateAccountBadRequest(AccountTestCase):
     def test_bad_request_error_is_raised(self):
-        expected_error_details = {
-            u'ModelState': {
-                u'account.url': [u'Invalid Url', u'Missing url parameter']
-            },
-            u'Message': u'The request is invalid.'
-        }
-
-        self.assertHTTPErrorIsRaised(
-            SitewitService().create_account, (
+        with self.assertRaises(HTTPServiceError) as e:
```

Comment:
It was changed because sometimes we've got `[u'Invalid Url', u'Missing url parameter']` and sometimes `[u'Missing url parameter', u'Invalid Url']`.  That's why the helper wasn't very useful here.